docs/source/advanced_usage.rst

```diff
@@ -52,15 +52,18 @@ concurrent context.
 
 Below is an example that demonstrates how to use pyQuil in a multithreading scenario:
 
+.. todo::
```

Comment:
This does not appear to be a valid rst/sphinx item and should just be a comment instead.

I find myself wishing for rustdoc's `norun` so that we can at least verify that types exist, etc.

Reply:
Fixed, but still a TODO